modules/dashboard/cmd/main.go

```diff
@@ -176,7 +210,10 @@ func main() {
 	flagset := cmd.Flags()
 	flagset.SortFlags = false
 	flagset.StringVarP(&cli.Config, "config", "c", "", "Path to configuration file (e.g. \"/etc/kubetail/dashboard.yaml\")")
-	flagset.StringP("addr", "a", ":8080", "Host address to bind to")
```

Comment:
Let's remove all of the addr-related flags altogether. You can already set these values easily with a config file or by using the "param" flag.